src/components/utils/OffchainSigner/OffchainSignerUtils.ts

```diff
@@ -0,0 +1,116 @@
+import { newLogger } from '@subsocial/utils'
+import axios, { AxiosRequestConfig } from 'axios'
+
+import { truncateAddress } from 'src/utils/storage'
+
+const log = newLogger('OffchainSignerRequests')
+
+import config from 'src/config'
+
+const { offchainSignerUrl } = config
+
+type AxiosResponse<T = any> = {
+  data: T
+  status: number
+}
+
+const setAuthOnRequest = (jwt: string) => {
+  try {
+    axios.interceptors.request.use(
+      async (config: AxiosRequestConfig) => {
+        config.headers = config.headers ?? {}
+
+        config.headers.Authorization = jwt
+
+        return config
+      },
+      error => {
+        return Promise.reject(error)
+      },
+    )
+  } catch (err) {
+    console.warn(err)
+  }
+}
+
+type SubmitSignedCallDataProps = {
+  data: any
+  jwt: string
+}
+
+export const isAccountOnboarded = (accountId: string) =>
+  localStorage.getItem(`df.onBoardingModalFinished:${truncateAddress(accountId)}`) === '1'
+
+export const submitSignedCallData = async ({ data, jwt }: SubmitSignedCallDataProps) => {
+  const signerEndpoint: OffchainSignerEndpoint = OffchainSignerEndpoint.SIGNER_SIGN
+  const method = 'POST' as Method
+  const dataPayload = {
+    unsigedCall: data,
```

Comment:
Looks like data it is HEX in all cases, so lets change `any` to string, and add comment?

Reply:
Changed from `any` to `string`

What do you mean by `add comment`?